### ISSUE TITLE
False error in SAM "Policies"

### ISSUE THREAD

**Issue by jashanbhullar** [ScottBrenner/cfn-lint-action#20]
This template gives an error
```
GenericLambdaFunction:
        Type: AWS::Serverless::Function
        Properties:
            CodeUri: path/to/code
            Policies:
                - S3ReadPolicy:
                      BucketName: !Ref BucketName
                - S3WritePolicy:
                      BucketName: !Ref SomeOtherBucket
                - AWSSecretsManagerGetSecretValuePolicy:
                      SecretArn: !Ref SomeSecretKey
            MemorySize: 2048
            Timeout: 600
```

Error:
```
Error transforming template: Resource with id [GenericLambdaFunction] is invalid. Policy at index 1 in the 'Policies' property is not valid
```

Couldn't find an resolution to this?
Can we disable it for some lines or something else can solve this?

**Comment by ScottBrenner** [comment]
Seems like an issue with the upstream/actual cfn-lint project, suggest opening an issue there.

**Comment by jashanbhullar** [comment]
@ScottBrenner thanks for the quick response. I think it's the version error, How can I check with `version` is the `cfn-lint` running in this action/

**Comment by jashanbhullar** [comment]
I checked using this 
```
with:
                  # template.yaml i.e. Main CFN template file
                  args: "input-template.yaml -v"
```
and it shows `0.28.1` while on my local I have `0.33.1` 
how can I upgrade the version in the action?

**Comment by ScottBrenner** [comment]
Which version of this action ate you using?  Suggest `scottbrenner/cfn-lint-action@master` as it will always have the most recent version of `cfn-lint`.

**Comment by jashanbhullar** [comment]
My bad, I was using `docker://scottbrenner/cfn-lint-action:latest, didn't notice it was updated.
